### PR TITLE
[ADD] Added account_credit_control_payment_details module

### PR DESCRIPTION
**Pull request by JayVora-SerpentCS**

### CODE REVIEW

File: account_credit_control_payment_details/README.rst [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+   :alt: Odoo Community Association
+   :target: https://odoo-community.org
+
+This module is maintained by the OCA.
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
+
+To contribute to this module, please visit http://odoo-community.org.
```
Comment (tafaRU) [inline account_credit_control_payment_details/README.rst]: s/http/https

File: account_credit_control_payment_details/tests/__init__.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Serpent Consulting Services Pvt. Ltd
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from . import test_credit_control_payment_details
```
Comment (tafaRU) [inline account_credit_control_payment_details/tests/__init__.py]: no newline

**Review by tafaRU — CHANGES_REQUESTED** [review]
flake8 error are unrelated --> OK
things left to do: fix last nitpicks and squash commits.

File: account_credit_control_payment_details/README.rst [hunk 3]
```diff
@@ -0,0 +1,69 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+======================================
+Account Credit Control Payment Details
+======================================
+
+For credit control payment details in PDF report, Go to ``Accounting > Periodical Processing > Credit
```
Comment (tafaRU) [inline account_credit_control_payment_details/README.rst]: @JayVora-SerpentCS, the module description and its usage description are mixed up! Could you please arrange them?

Thank you in advance.

Comment (JayVora-SerpentCS) [reply account_credit_control_payment_details/README.rst]: Processed.

**Review by tafaRU — APPROVED** [review]

**Comment by JayVora-SerpentCS** [comment]
Should be merged now!

**Comment by JayVora-SerpentCS** [comment]
Thanks.

**Pull request merged**